polygeist/tools/cgeist/driver.cc

```diff
@@ -717,26 +709,13 @@ static int createAndExecutePassPipeline(
   return 0;
 }
 
-static llvm::Optional<llvm::OptimizationLevel> optionsToOptimizationLevel() {
-  if (Opt0)
-    return llvm::OptimizationLevel::O0;
-  if (Opt1)
-    return llvm::OptimizationLevel::O1;
-  if (Opt2)
-    return llvm::OptimizationLevel::O2;
-  if (Opt3)
-    return llvm::OptimizationLevel::O3;
-  // Not explicit optimization level not handled by cgeist
-  return llvm::None;
-}
-
 /// Run optimization pipeline in LLVM module.
 ///
 /// Just run default pipelines for now.
-static void runOptimizationPipeline(llvm::Module &module) {
-  const llvm::Optional<llvm::OptimizationLevel> optLevel =
-      optionsToOptimizationLevel();
-  if (!optLevel || optLevel->getSpeedupLevel() == 0) {
+static void runOptimizationPipeline(
+    llvm::Module &module,
+    const llvm::Optional<llvm::OptimizationLevel> &OptimizationLevel) {
+  if (!OptimizationLevel || *OptimizationLevel == llvm::OptimizationLevel::O0) {
```

Comment:
Same comment as above about `!OptimizationLevel`.

Reply:
I'll run `-O2` if no option is provided.